core/migrations/0030_auto_20181015_1052.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.1.2 on 2018-10-15 10:52
```

Comment:
IMHO, I would suggest we have this migration and `0031_auto_20181015_1424.py` as one migration